.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Coverage-test
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
I think a good convention would be to set workflow_dispatch to `{}`. Otherwise, this looks incomplete (despite it is completed). Do you mind looking this up?